config/schedule.rb

```diff
@@ -50,6 +50,12 @@ def local(time)
   end
 end
 
+every :day, at: local("02:00 pm"), roles: [:cron] do
+  if CountryConfig.current_country?("Bangladesh") && SimpleServer.env.production?
```

Comment:
Conventionally, we're keeping checks inside the every block. I'm wondering if we should put every block inside the condition checks? That way, each country's crontab files will only have rake tasks relevant for that environment. Something like this, maybe?

```ruby
if CountryConfig.current_country?("Bangladesh") && SimpleServer.env.production?
  every :day, at: local("02:00 pm"), roles: [:cron] do
    rake "alpha_sms:check_balance"
  end
end
```

Reply:
Looks like this is happening regardless. The task only gets converted to cron syntax if the condition matches.

Reply:
Hmm, that's nice then. PR looks good to be merged.